.gitmodules

```diff
@@ -31,3 +31,6 @@
 [submodule "lib/sdl/SDL_image"]
 	path = lib/sdl/SDL2_image
 	url = https://github.com/SDL-mirror/SDL_image.git
+[submodule "lib/sdl/SDL2_gfx"]
+	path = lib/sdl/SDL2_gfx
```

Comment:
I'm not sure if this belongs into lib/sdl.

This library is not an official SDL project - the other projects in that folder are either official or done by SDL organization members and promoted by SDL.